README.md

```diff
@@ -1,2 +1,6 @@
 # cell_counting
 We create machine learning algorithm to count cells in the microscopy photo
+
+
+## https://www.jetbrains.com/pycharm/download/#section=windows
```

Comment:
what if I want to install it on linux? will this url work?

Reply:
nope, sorry

https://www.jetbrains.com/pycharm/download/

is the right one?

Reply:
yes :)

Reply:
<3